www/include/configuration/configObject/servicegroup/getServiceXml.php

```diff
@@ -49,32 +49,32 @@
 $centreon = $_SESSION['centreon'];
 $db = new CentreonDB();
 
-$hostId = $_POST['host_id'];
+$hostId = filter_var($_POST['host_id'], FILTER_VALIDATE_INT);
 $acl = $centreon->user->access;
 $xml = new CentreonXML();
 $xml->startElement("response");
-if ($hostId != "") {
+if ($hostId) {
```

Comment:
Logically incorrect, what's happen if id = 0 ?
```suggestion
if ($hostId !== false) {
```

Reply:
$hostId !== false && $hostId > 0